### DISCUSSION TITLE
gitignore contents for multiple users

### DISCUSSION THREAD

**Discussion by mikevang** [brefphp/bref#1165] [Support] [answered]
Should the `.serverless` folder be added to the `gitignore` file? We have multiple devs working on a project and I'm unsure as to whether or not the `.serverless` folder contains info that would be helpful when code is passed off to other devs. Or does the info in that folder get regenerated with each deploy regardless? Thanks.

**Comment by mnapoli** [comment, marked as answer]
It should be gitignored, yes.

**Reply by mikevang** [reply]
Thanks
